custom_components/google_home/models.py

```diff
@@ -35,10 +35,17 @@ def __init__(
         self.auth_token = auth_token
         self.ip_address = ip_address
         self.hardware = hardware
-        self.available = True
         self._timers: List[GoogleHomeTimer] = []
         self._alarms: List[GoogleHomeAlarm] = []
 
+    def has_alarms(self) -> bool:
+        """Returns whether or not device has alarms"""
+        return self._alarms != []
```

Comment:
I'd rather use this:
```suggestion
        return len(self._alarms) > 0
```